react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.stories.tsx

```diff
@@ -64,49 +64,80 @@ DashedWells.parameters = {
     },
 };
 
-export const CustomDashedWells = Template.bind({});
-CustomDashedWells.args = {
+export const CustomColoredWells = Template.bind({});
+CustomColoredWells.args = {
     ...defaultProps,
     layers: [
         {
             ...defaultProps.layers[0],
-            dashed: [10, 3],
+            lineStyle: { color: [255, 0, 0, 255], dash: [10, 3] },
             refine: false,
             outline: false,
         },
     ],
 };
-CustomDashedWells.parameters = {
+CustomColoredWells.parameters = {
     docs: {
         description: {
-            story: "Volve wells example with dashed well trajectories, with custom style.",
+            story: "Volve wells example with dashed style and red trajectories, with custom style.",
         },
         inlineStories: false,
         iframeHeight: 500,
     },
 };
 
-function callback(object: Record<string, Record<string, unknown>>) {
-    if (object["properties"]["name"] == "15/9-F-11 B") return [5, 3];
-    return [0, 0];
+function colorCallback(object: Record<string, Record<string, unknown>>) {
+    if ((object["properties"]["name"] as string).match("15/9-19"))
+        return object["properties"]["color"];
+    else if ((object["properties"]["name"] as string).match("15/9-F"))
+        return [0, 0, 0, 0];
+    else return [0, 0, 0, 255];
 }
 
-export const CallbackDashedWells = Template.bind({});
-CallbackDashedWells.args = {
+function dashCallback(object: Record<string, Record<string, unknown>>) {
```

Comment:
For the callback example, I think it would be nice to show all three use cases, ie. both hidden, dashed and unstyled. In this example all shown wells are dashed.